AlexeyShparun/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def uploading_data
+  data = File.new('data.txt')
+  data.read
+end
+
+def mean(town, string)
+  return -1 unless string.include?(town)
```

Comment:
1) what does `return -1` mean for error handling? 
2)it's better to move city validation to corresponding `city_search` method

same comments apply to this line in `variance` method